Compilation error in WebSocketSessionContext with GraalVM SDK 22

When compiling with the GraalVM JDK 22, I get the following error:

> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.12.1:compile (default-compile) on project quarkus-websockets-next: Compilation failure
> [ERROR] /home/gsmet/git/quarkus/extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketSessionContext.java:[228,40] variable IS_VALID might not have been initialized

I'm not entirely sure if it's a problem with the code or with the GraalVM JDK:

https://github.com/quarkusio/quarkus/blob/b870f323fc902285dbc67ed4b64f46b73bfe4b83/extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketSessionContext.java#L228-L236

(discovered that by mistake as I forgot resetting my JDK after testing something)

/cc @zakkak @mkouba

--- a/extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketSessionContext.java
+++ b/extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketSessionContext.java
@@ -215,7 +215,7 @@ public class WebSocketSessionContext implements ManagedContext {
         });
     }
 
-    class SessionContextState implements ContextState {
+    static class SessionContextState implements ContextState {
 
         // Using 0 as default value enable removing an initialization
         // in the constructor, piggybacking on the default value.

